paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlSchema.java

```diff
@@ -36,7 +38,7 @@ public class MySqlSchema {
     private final String databaseName;
     private final String tableName;
 
-    private final LinkedHashMap<String, DataType> fields;
+    private final Map<String, Tuple3<String, DataType, String>> fields;
```

Comment:
Why use Map?

Reply:
> Why use Map instead of LinkedHashMap?

Line 50 is directly initialized into LinkedHashMap.